src/agent/src/image_rpc.rs

```diff
@@ -113,10 +99,12 @@ impl ImageService {
             .arg(&target_path_bundle)
             .status()?;
 
-        ensure!(status.success(), "failed to unpack image: {:?}", status);
+        if !status.success() {
+            return Err(anyhow!(format!("failed to unpack image: {:?}", status)));
```

Comment:
You don't need `format!()` if you're using `anyhow!()`.

Reply:
Again, bad merge with the alibaba image refactor, so gone back to their code. Thanks for spotting it